central/graphql/resolvers/clusters.go

```diff
@@ -953,3 +956,41 @@ func (resolver *clusterResolver) UnusedVarSink(_ context.Context, _ RawQuery) *i
 func (resolver *orchestratorMetadataResolver) OpenShiftVersion() (string, error) {
 	return resolver.data.GetOpenshiftVersion(), nil
 }
+
+// PlatformCVECountByType returns resolver to get number of platform CVEs by CVE type
```

Comment:
For my education: What does a resolver do?

Reply:
If there is a graphQL query like

```
cluster(id) {
    PlatformCVEsCountByType {
        Kubernetes
        Openshift
        Istio
    }
}
```

Here Kubernetes, Openshift and Istio are of `integer` type. But `cluster(id)` returns a complex type on which you can query other things like `PlatformCVEsCountByType`. Similarly `PlatformCVEsCountByType` is also a complex type on which you can query other things until you reach a scalar type like int or string. In graphQL implementations in our code base, we call these complex types resolvers. But they are just interfaces really.